### DISCUSSION TITLE
Termostato TS0601 riconosciuto ma con solo 2 entità disabilitate

### DISCUSSION THREAD

**Discussion by maurominella** [indomus/forum#1496] [HUB: Home Assistant]
Ho fatto rilevare una valvola termostatica zigbee da ZHA, che ha riconosciuto un device di tipo TS0601 ma non del tutto bene: infatti le associa 2 entità numeriche che sono pure disabilitate, invece delle tre classiche entità (temperatura-umidità-termostato) che mi permetterebbero di associare la valvola ad una entità termostato:
![image](https://user-images.githubusercontent.com/32428960/153715983-65704386-9927-4b22-a28d-f776ef0742a2.png)

Ora posso anche abilitare queste eneità, e in questo caso mi rilevano i valori (176 e -56 come si vede nell'immagine sopra), tuttavia non riesco a capire che cosa rappresentino questi numeri. Ma soprattutto ho bisogno del sensore che mi permette di regolare la temperatura associandolo ad un template climate. Se lo forzo, ottengo un errore.
Questa la device signature della valvola (che è questa: it.aliexpress.com/item/1005003246227640.html)
![image](https://user-images.githubusercontent.com/32428960/153716071-d19e8daa-4bac-49a4-a934-b197bc14ea8c.png)

**Comment by snakuzzo** [comment]
Quelli sono i cluster Zigbee ...ma "a te" non servono a niente quelle info
Ti consiglierei di aprire una issue su [core](https://github.com/home-assistant/core/issues/)...noi possiamo fare ben poco

**Reply by maurominella** [reply]
ok grazie. [Ho fatto subito](https://github.com/home-assistant/core/issues/66400), vediamo che succede perchè ho comperato queste valvole che dal punto di vista hardware sono le migliori, precise nella rilevazione della temperatura, utilizzo NON touch come altre molto meno buone, lunga durata batteria grazie a pochi fronzoli inutili.
E funzionano perfettamente con Smart Life, da cui escluderei un problema nelle stesse valvole.

Speravo si trattasse solo di forzare una corretta configurazione nei file di Home Assistant.

**Reply by snakuzzo** [reply]
Io fossi in te passerei a Zigbee2mqtt. Lì mi sa che sono pienamente supportate

**Reply by maurominella** [reply]
grazie @snakuzzo ho approfondito il tuo suggerimento. Però mi sfuggono un paio di concetti chiave:

Premesso che io sto utilizzando:
- Home Assistant installato su un sistema operativo Linux standard (in realtà è una macchina virtuale) su cui appunto ho installato uno per uno tutti i componenti di home assistant, che funziona benissimo
- il sonoff zigbee bridge patchato tasmota che a una parte "parla zigbee" permettendomi di agganciare qualsiasi dispositivo zigbee, e dall'altra si connette alla wifi di casa così da porterlo vedere tramite il suo indirizzo IP sulla porta 8888. 
- l'integration service ZHA che si connette al bridge usando la rete con una configurazione tipo socket://<bridge_ip_address:8888
 
La mia domanda è: come si posiziona questa architettura rispetto a quella che più volte ~~tu hai indicato~~ avete indicato nei ~~tuoi~~ vostri articoli?
![image](https://user-images.githubusercontent.com/32428960/153744213-1a9b45e0-5c5d-4b19-8050-17d4e4e77769.png)

Tu parli sempre di "antenna" tipo dongle USB che va configurata sul server, ma nel mio attuale caso io non ho dovuto configurare nulla sul server o sull'home assistant, ma ho semplicemente configurato l'integration ZHA con l'indirizzo del tasmota bridge. Mi verrebbe quindi da dire che il mio Tasmota Bridge si pone come sostituto dell'ultimo elemento blu a destra (zigbee2mqtt), sebbene non sono sicuro che "parli mqtt" verso sinistra. In altre parole, servono comunque i due componenti broker e zigbee2mqtt in aggiunta al mio bridge, e se sì come aggancio lo zigbee2mqtt al tasmota, che si espone sulla porta 8888 del socket ip?

Grazie per il chiarimento

**Reply by snakuzzo** [reply]
Risolveresti tutto non usando il zbbridge ma usando una soluzione come la z-bee duo di @gio-dot che puoi piazzare dove ti pare...indipendentemente dal server.

Sul server ti occorrerebbe solo installare il broker mqtt e zigber2mqtt nella cui configurazione gli passerai l'ip della chiavetta per parlarci via seriale su TCP.

Es. `http://192.168.1.22:8888`

**Reply by maurominella** [reply]
In realtà ora è davvero comodo perché questo sonoff bridge è grande e pesante quanto una scatola di fiammiferi (molto meno di una Raspberry) che si connette alla wifi di casa e quindi è già separato dal server, è alimentato a pile o mini-USB e non ha bisogno di nient'altro. Nel mio caso il server non è nemmeno in casa, ma su una macchina virtuale Linux sul cloud di Microsoft Azure che si collega al bridge attraverso l'indirizzo pubblico del mio router di casa, su cui ho mappato la porta interna 8888 (naturalmente verso l'esterno uso una porta diversa, tipo socket://casamia.dyndns.org:98765). La scansione dei dispositivi zigbee io l'ho sempre fatta collegato ad un home assistant che gira sul server di un datacenter di Dublino, così posso vedere i miei device mentre sono in casa o al lavoro, in treno sul cellulare o, per la casa al mare, prima di partire da Milano per accendere il riscaldamento d'inverno. Ora non vedo problema a sostituire il singolo componente ZHA con la coppia zigbee2mqtt + mosquitto, però se ho capito bene in questo caso dovrei rinunciare anche all'hardware sonoff, corretto?

**Reply by snakuzzo** [reply]
Non dovresti mettere un raspberry. La chiavetta di giovanni pesa pure meno di un zbbridge ed è indipendente e la useresti come ora usi il zbbridge.
Comunque non mi pare tu possa usare il zbbridge cpn zigbee2mqtt.
Ad ogni modo ti consiglierei di aspettare novità sulla issue che hai riscontrato e segnalato

**Reply by indomus** [reply]
> La mia domanda è: come si posiziona questa architettura rispetto a quella che più volte tu hai indicato nei tuoi articoli? 

Mauro, per inciso gli articoli non sono scritti da Diego - per quanto sia un (prezioso) collaboratore di questa community.

**Comment by snakuzzo** [comment]
Non so se possa interessare...
https://github.com/Koenkk/zigbee-herdsman-converters/pull/3867
